spec/line/bot/rich_menu_spec.rb

```diff
@@ -95,6 +95,14 @@
       .with(body: JSON.parse(RICH_MENU_CONTENT).to_json)
   end
 
+  it 'validates a rich menu' do
+    uri_template = Addressable::Template.new Line::Bot::API::DEFAULT_ENDPOINT + '/bot/richmenu/validate'
+    stub_request(:post, uri_template).to_return(body: '{}', status: 200)
+
+    client.validate_rich_menu(JSON.parse(RICH_MENU_CONTENT))
+    expect(WebMock).to have_requested(:post, Line::Bot::API::DEFAULT_ENDPOINT + '/bot/richmenu/validate')
+  end
```

Comment:
[must fix]

This should be changed according to the change of the method name.

```suggestion
  it 'validates a rich menu object' do
    uri_template = Addressable::Template.new Line::Bot::API::DEFAULT_ENDPOINT + '/bot/richmenu/validate'
    stub_request(:post, uri_template).to_return(body: '{}', status: 200)

    client.validate_rich_menu_object(JSON.parse(RICH_MENU_CONTENT))
    expect(WebMock).to have_requested(:post, Line::Bot::API::DEFAULT_ENDPOINT + '/bot/richmenu/validate')
  end
```

Reply:
Thank you for your review.
I understand your comments and I will fix it later.